docs/configuration/configuration.md

```diff
@@ -134,7 +133,7 @@ For example, the `--cookie-secret` flag becomes `OAUTH2_PROXY_COOKIE_SECRET` and
 the `--set-authorization-header` flag becomes `OAUTH2_PROXY_SET_AUTHORIZATION_HEADER`.
 
 
-## Logging Configuration
+## <a anchor="logging-configuration"></a>Logging Configuration
```

Comment:
I don't think this is necessary to set explicitly, it already works out of the box no?https://pusher.github.io/oauth2_proxy/docs/configuration#logging-configuration

Reply:
It looks like you're right. I saw that there was an <a> tag on the "Configuring for use with the Nginx `auth_request` directive" heading and my intention was to be consistent with what was being don in the doc already. Looks like my tag wasn't even setup right, should have been `< a name` instead of `<a anchor`.

I've removed that one and the one on the "Configuring for use with the Nginx `auth_request` directive" heading and confirmed that jekyll still renders the html correctly.

Reply:
I just noticed that the `<a>` tag next to the "Configuring for use with the Nginx auth_request directive" header was to provide a shorter link target for that header, so I've pushed another change to put that one back. Oops.